### ISSUE TITLE
NullPointerException in DecryptActivity

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#1391]
From G Play
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.sufficientlysecure.keychain/org.sufficientlysecure.keychain.ui.DecryptActivity}: java.lang.NullPointerException
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2198)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2257)
at android.app.ActivityThread.access$800(ActivityThread.java:139)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1210)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:136)
at android.app.ActivityThread.main(ActivityThread.java:5097)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:515)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at org.sufficientlysecure.keychain.ui.DecryptActivity.handleActions(DecryptActivity.java:127)
at org.sufficientlysecure.keychain.ui.DecryptActivity.onCreate(DecryptActivity.java:61)
at android.app.Activity.performCreate(Activity.java:5248)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1110)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2162)
... 11 more
```

**Comment by Valodim** [comment]
this one should already be fixed in 27441c7d63175255d1a50e96ad6418592666a3db

**Comment by Valodim** [comment]
also made other locations where the clipboard systemservice is called resilient against nullptrs
